### DISCUSSION TITLE
Vectric post processor?

### DISCUSSION THREAD

**Discussion by stevempotter** [grblHAL/core#49] [Q&A]
I have finished the upgrade of my MPCNC to grblHAL on Phil Barrett's Teensy BOB, about to start testing! Is there a post processor for grblHAL that can be used in VcarvePro (Vectric)?
I have seen a grbl one, but I imagine it does not take advantage of speed improvements in grblHAL hardware.

**Comment by terjeio** [comment]
> I have seen a grbl one, but I imagine it does not take advantage of speed improvements in grblHAL hardware.

The grbl one you have seen is fine, the only improvement would be to add tool change support?

If you are using ioSender you may configure VCarve to use [direct output](https://github.com/terjeio/ioSender/wiki/Vectric-Direct-Output), I find it very convenient.
